examples/plot_tract_profile.py

```diff
@@ -124,10 +124,9 @@
     for bundle in bundles:
         for idx, roi in enumerate(bundles[bundle]['ROIs']):
             if bundles[bundle]['rules'][idx]:
-                warped_roi = patch_up_roi(
-                    mapping.transform_inverse(
-                        roi.get_fdata().astype(np.float32),
-                        interpolation='linear'),
+                warped_roi = auv.transform_inverse_roi(
```

Comment:
You currently have: 

```
from AFQ.utils.volume import transform_inverse_roi

...

warped_roi = auv.transform_inverse_roi(...)

```

I think you want either: 


```
import AFQ.utils.volume as auv
...

warped_roi = auv.transform_inverse_roi(...)

```

or: 

```
from AFQ.utils.volume import transform_inverse_roi

...

warped_roi = transform_inverse_roi(...)

```

Reply:
Good catch! The convention in the examples seems to be to do the latter, while the former is used elsewhere in the code.